project/threads/models.py

```diff
@@ -1,3 +1,543 @@
+import os
```

Comment:
As @gorkemarslan mentions, these changes should be in a separate pull request, since they are not related to the CodeQL workflow.